crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -180,6 +180,20 @@ def save
       true
     rescue StandardError => e
       @logger.error("Exception during saving the status file: #{e.message}")
+      # clear old errors to make sure there is only one type of that error present
+      # this is necessary to query for such an error through the :async_save_fail before_filter
+      # in the UpgradeController
+      error = Api::Error.find_by(
```

Comment:
Do we have access to this from all usages of progress library (e.g. from crowbar-init)?